docs/field-details.asciidoc

```diff
@@ -5290,6 +5290,731 @@ example: `3.2.4`
 
 // ===============================================================
 
+|=====
+
+[[ecs-sip]]
+=== SIP Fields
+
+These fields are used to describe SIPand AD based conversations between endpoints and servers, as well as direct communications sessions between endpoints.
```

Comment:
```suggestion
These fields are used to describe SIP and AD based conversations between endpoints and servers, as well as direct communications sessions between endpoints.
```

Reply:
committed!  good catch

Reply:
actually trying to figure out the "SIP and AD" - AD as in active directory?  (Yes I wrote this, and no I cannot figure out what that might be in reference to :)

Reply:
haha I assumed it had to do with Skype for business or similar https://docs.microsoft.com/en-us/skypeforbusiness/plan-your-deployment/security/active-directory-domain-services